README.md

```diff
@@ -20,6 +20,7 @@
 [![codecov](https://codecov.io/gh/proshin-roman/finapi-java-client/branch/master/graph/badge.svg)](https://codecov.io/gh/proshin-roman/finapi-java-client)
 [![codebeat badge](https://codebeat.co/badges/907cb4b8-0f65-446a-a874-9ece228ab579)](https://codebeat.co/projects/github-com-proshin-roman-finapi-java-client-master)
 [![SonarCloud](https://sonarcloud.io/api/project_badges/measure?project=org.proshin%3Afinapi-java-client&metric=alert_status)](https://sonarcloud.io/dashboard?id=org.proshin%3Afinapi-java-client)
+[![Coverage Status](https://coveralls.io/repos/github/proshin-roman/finapi-java-client/badge.svg?branch=coverall-integration)](https://coveralls.io/github/proshin-roman/finapi-java-client?branch=master)
```

Comment:
Not sure exactly what the order should be. Is this ok? Or should this be right after `codecov`, as they both return a code coverage percentage?